polyfill/test/thorough/support.mjs

```diff
@@ -0,0 +1,361 @@
+// == Exhaustive / snapshot testing framework ==
+//
+// To use: Import this file and use its API in your test file. See
+// datedifference.mjs for an example.
+//
+// When you execute that file with `node`, by default it will test the Temporal
+// polyfill.
+// To update snapshots (i.e., matchSnapshot assertions will not fail and the
+// snapshot file will be created or updated to match the current state of the
+// Temporal polyfill) pass the -u argument.
+//
+// You can also test built-in implementations in other interpreters. It's
+// easiest to install them using esvu if possible. You should be able to run
+// this file with any JS interpreter, although it must be modern enough to
+// support JSON imports. Snapshot updating is not available with other
+// interpreters, since it requires Node's process and fs/promises modules.
```

Comment:
Couldn't this be addressed by writing snapshot data to stdout (e.g., via `print`) for redirection outside the process? There's no need to actually implement it, although maybe updating the comment to that effect would be nice in case this becomes important.